dotcom-rendering/src/components/ScrollableCarousel.tsx

```diff
@@ -77,16 +77,6 @@ const containerStyles = css`
 	}
 	${from.leftCol} {
 		margin-left: 0;
-		::before {
```

Comment:
There are some related styles on lines 121–125 to position the pseudo element that can also be removed 🗑️